src/focus.service.ts

```diff
@@ -521,11 +521,10 @@ export class FocusService {
       return false;
     }
 
-    if (el.tabIndex > -1) {
-      return true;
+    if (el.tabIndex < 0) {
+      return false;
     }
```

Comment:
I have no idea. I think Edge is having it default to 0 though: https://msdn.microsoft.com/en-us/library/windows/apps/hh780161.aspx